locations/spiders/press_shop_be.py

```
@@ -1,34 +1,64 @@
+                blob += json.loads(segment)
+        return parse_js_object(blob[blob.find('"shops":[') + len('"shops":') :])
+
+    def pre_process_data(self, feature: dict) -> None:
+        feature.update(feature.pop("latlong"))
+        feature["street_address"] = feature.pop("address", None)
+        feature["postcode"] = feature.pop("postal", None)
+        feature["state"] = feature.pop("region", None)
+
+    def post_process_item(self, item: Feature, response: TextResponse, feature: dict) -> Iterable[Feature]:
+        title = (item.pop("name") or "").strip()
+        for prefix, (attributes, category) in self.brands.items():
```

Ruff (0.16.1)

[warning] Loop control variable `attributes` not used within loop body

(B007)